programmers/문자열-내-p와-y의-갯수.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var answer = true;
+    var pcount = 0;
+    var ycount = 0;
+    for (var i in s){
+        if (s[i] == 'p' || s[i] == 'P'){
+            pcount++;
+        }
+        else if (s[i] == 'y' || s[i] == 'Y'){
+            ycount++;
```

Comment:
개인적으로 사전에 입력값 s를 toLowerCase()와 같은 메서드로 전체를 소문자로 바꾸고 시작했으면 if문이 좀 더 간결해질 수 있을 것 같습니다. toLowerCase()도 되고 toUpperCase()도 괜찮습니다.

```suggestion
  s = s.toLowerCase();
        if (s[i] === 'p'){
            pcount++;
        }
        else if (s[i] === 'y'){
            ycount++;
```